converter/datamodel.py

```diff
@@ -294,15 +294,15 @@ def from_magetab(cls, study_info):
 
         contact_terms = get_controlled_vocabulary("contact_terms")
         contacts_raw = study_info.get("contacts", [])
-        contacts = [Contact(c.get(contact_terms["personfirstname"]),
-                            c.get(contact_terms["personlastname"]),
-                            c.get(contact_terms["personemail"]),
-                            c.get(contact_terms["personaffiliation"]),
-                            c.get(contact_terms["personaddress"]),
-                            c.get(contact_terms["personphone"]),
-                            c.get(contact_terms["personroles"]),
-                            c.get(contact_terms["personmidinitials"]),
-                            c.get(contact_terms["personfax"])
+        contacts = [Contact(c.get(contact_terms["Person First Name"]),
```

Comment:
If Contact is an object, you could have a constructor that understands this contact_terms dictionary and leave that mapping logic there, making the flow easier here.

Reply:
or an auxiliar factory class to do that.

Reply:
Yes, I was thinking about how this could be solved more elegantly but couldn't come up with a good idea. So I would love to discuss this with you and learn about factory classes :)